torch/fx/graph.py

```diff
@@ -499,11 +499,11 @@ def emit_node(node : Node):
                 meta_val = node.meta.get('val', node.meta.get('tensor_meta', None))
 
                 if isinstance(meta_val, FakeTensor):
-                    maybe_type_annotation = f': {dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}'
+                    maybe_type_annotation = f': {dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}, {stringify_shape(meta_val.stride())}'
```

Comment:
I would suggest not to include stride by default.

Reply:
yeah you probably shouldn't put this in this PR